README.md

```diff
@@ -172,15 +172,27 @@ cd chisel3
 sbt compile
 ```
 
-In order to run the following unit tests, make sure you have Verilator installed and on your `PATH`
-(you can check this by running `which verilator`).
+In order to run the following unit tests, you will need several tools on your `PATH`, namely
+[verilator](https://www.veripool.org/verilator/),
+[yosys](http://www.clifford.at/yosys/),
+[espresso](http://web.eecs.umich.edu/~ksewell/espresso/),
```

Comment:
How about https://github.com/chipsalliance/espresso since we have re-hosted it?

Reply:
And I believe this link was a tutorial, the original official link was https://ptolemy.berkeley.edu/projects/embedded/pubs/downloads/espresso/index.htm